package/MDAnalysis/analysis/rdf.py

```diff
@@ -645,9 +645,17 @@ def _single_frame(self):
                 backend=self.backend,
             )
 
-            for j, (idx1, idx2) in enumerate(pairs):
-                count, _ = np.histogram(dist[j], **self.rdf_settings)
-                self.results.count[i][idx1, idx2, :] += count
+            bins = self.rdf_settings["bins"]
```

Comment:
Can we rename to `nbins` to make clear that these are simple equi-distant bins?
```suggestion
            nbins = self.rdf_settings["bins"]
```
and then change below.